hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestBulkDeleteCommand.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.hadoop.fs.shell;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.LinkedList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestBulkDeleteCommand {
```

Comment:
can we not add the actual tests to delete file?

Reply:
This is a FS layer level change, ideally this will work even if HADOOP_OPTIONAL_TOOLS is not pushed out to the command line. Or you mean something that creates and deletes files on the local fileSystem? Like a test for `rm -r`?

Reply:
yes I meant the local FS.